docs/source/notes/autograd.rst

```diff
@@ -119,3 +119,90 @@ a version counter of their containing Tensor is saved as well. Once you access
 an error is raised. This ensures that if you're using in-place
 functions and not seeing any errors, you can be sure that the computed
 gradients are correct.
+
+
+Multithreaded Autograd
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The autograd engine allows the caller thread to drive the backward execution, user
```

Comment:
This might be a bit to close to the history of the engine. While this page is supposed to described the current state I think. I would suggest something closer to:
The autograd engine is responsible for running all the ops necessary to compute the backward pass.
This section will describe all the details that can help you make the best use of it in a multithreaded environment (this is relevant only for pytorch 1.6+ as the behavior in previous version was different).